src/pages/bus/shuttle/index.tsx

```
@@ -1,21 +1,20 @@
+import { dehydrate, QueryClient, useQuery, useSuspenseQuery } from '@tanstack/react-query';
+import { busQueries } from 'api/bus/queries';
 import InformationIcon from 'assets/svg/Bus/info-gray.svg';
 import RightArrow from 'assets/svg/right-arrow.svg';
 import BusCoursePage, { useBusCourse } from 'components/Bus/BusCoursePage';
 import InfoFooter from 'components/Bus/BusCoursePage/components/InfoFooter';
 import { ShuttleCategoryTabs } from 'components/Bus/BusCoursePage/components/ShuttleCategoryTabs';
 import useBusPrefetch from 'components/Bus/BusCoursePage/hooks/useBusPrefetch';
-import { useClientShuttleTimetable } from 'components/Bus/BusCoursePage/hooks/useBusTimetable';
-import useShuttleCourse from 'components/Bus/BusCoursePage/hooks/useShuttleCourse';
 import { SSRLayout } from 'components/layout';
 import dayjs from 'dayjs';
```

ESLint

[error] `dayjs` import should occur before import of `api/bus/queries`

(import/order)